### PR TITLE
[ENH] Add interface to SPM realign_unwarp 

### PR DESCRIPTION
**Pull request by qtabs**
Adds am interface to SPM's realing & unwarp function.

### CODE REVIEW

File: nipype/interfaces/spm/preprocess.py [hunk 1]
```diff
@@ -437,6 +437,251 @@ def _list_outputs(self):
...
+    of time series. Function accepts only one single field map. If in_files is a 
+    list of files they will be treated as separate sessions but associated to the
+    same fieldmap.
+
+    http://www.fil.ion.ucl.ac.uk/spm/doc/manual.pdf#page=31
+
+    Examples
+    --------
+
+    >>> import nipype.interfaces.spm as spm
```
Comment (mgxd) [inline nipype/interfaces/spm/preprocess.py]: you'll have to add `RealignUnwarp` to https://github.com/nipy/nipype/blob/7ea4f1bd3e99a276ea99423a2534a68556e036c7/nipype/interfaces/spm/__init__.py#L8-L10
in order to import this way

File: nipype/interfaces/spm/preprocess.py [hunk 2]
```diff
@@ -437,6 +437,251 @@ def _list_outputs(self):
...
+    >>> realignUnwarp = spm.RealignUnwarp()
+    >>> realignUnwarp.inputs.in_files = ['func-run01.nii', func-run02.nii']
+    >>> realignUnwarp.inputs.phase_map = 'phasemap.vdm'
```
Comment (mgxd) [inline nipype/interfaces/spm/preprocess.py]: IIRC, these files must exist to satisfy the traits requirements - we have [quite a bit included in the repo already](https://github.com/nipy/nipype/tree/master/nipype/testing/data), but for the sake of interface clarity if you cannot reuse any existing files, feel free to add necessary (empty) files

**Comment by effigies** [comment]
Hi @qtabs, do you have time to finish this up this week?

**Comment by qtabs** [comment]
Thanks a lot for the help! :) I'm sorry I didn't reply any earlier, I left for vacation right after the request and couldn't check my mailbox until now. I'll try to implement your suggestions and run the tests again next week :).

**Comment by effigies** [comment]
Hi @qtabs. Please let us know if you need anything from us. Just a heads up that we'll try to release on August 27, so it'd be great to have this within the next two weeks (by August 20).

**Comment by qtabs** [comment]
Hi guys! I tried to fix the problems you hinted above but I still get some errors. I think (sorry, it's my first time dealing with automated tests!) the tests are failing to load the new module:
------------
597     >>> realignUnwarp = spm.RealignUnwarp()
UNEXPECTED EXCEPTION: AttributeError("'module' object has no attribute 'RealignUnwarp'",)
------------
but I have added RealignUnwarp to __init__.py as suggested by @mgxd, I'm not sure what's what I'm missing! O:)

Thanks a lot for your help!

**Comment by effigies** [comment]
> ```
> 597     >>> realignUnwarp = spm.RealignUnwarp()
> UNEXPECTED EXCEPTION: AttributeError("'module' object has no attribute 'RealignUnwarp'",)
> ```

I'm not seeing this issue in the logs. I am seeing:

```
NameError: name 'InputMultiObject' is not defined
```

Which just means you need to import `InputMultiObject` at the top of the file, or revert to `InputMultiPath`.

You should also merge master to fix the `BIDSDataGrabber` issue.

**Comment by effigies** [comment]
Merged master for you. If this passes it'll make it into 1.1.2 (we need to do an emergency release). If not, we'll push it off to 1.1.3 next month.

For reference I did the following:

1) Add your repository as a `qtabs` remote and the main repository as `upstream`.

```Shell
git remote add upstream git@github.com:nipy/nipype.git
git remote add qtabs git@github.com:qtabs/nipype.git
```

(You wouldn't need to add your own.)

2) Update all remotes to the latest state:

```Shell
git fetch --all
```

3) Check out your branch, merge `upstream/master` and push:

```Shell
git checkout add addSPMrealingUnwarp
git merge upstream/master
git push
```

**Comment by qtabs** [comment]
Thanks a lot @effigies! I'm trying my best but I'm still quite new to all this stuff, sorry for my noobie incompetence! O:)

**Comment by effigies** [comment]
No worries. I'd usually try to be a little more helpful and give newbies a chance to practice, but with a shortened release window, I figured we could squeeze this one in. I hope you keep your eyes out for some more practice opportunities. :-)

**Comment by effigies** [comment]
Well, the latest failures aren't related to this PR. I'm fixing them in #2679.

**Pull request merged**